app/templates/client/styles/commons.scss

```diff
@@ -1,6 +1,58 @@
-@import 'globals';
-@import '~react-toolbox/lib/commons';
+@import "../../node_modules/modern-normalize/modern-normalize.css";
```

Comment:
What does this do vs all of the stuff below?

Reply:
The stuff below is basically a typeset base. Nothing special, but sizing some of the base elements.

Normalize does a few things. It is very much like `normalize.css`, but "modern".  The most important one being `box-sizing: border-box` which is the thing I have been missing in my css life. Short summary is:
```
Normalizes styles for a wide range of elements.
Corrects bugs and common browser inconsistencies.
```